src/xmipp/libraries/interface/frm.cpp

```diff
@@ -28,6 +28,32 @@
 #include <core/geometry.h>
 #include <core/transformations.h>
 
+void print_traceback(){
```

Comment:
You should put this method in a new file "utils" or in another with general functions in order to use it later

Reply:
I agree. @dmaluenda , @cossorzano , do we have a good place for this?

Reply:
I have extracted this functionality to python_utils.cpp/h